src/dragDrop.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-unused-vars, no-use-before-define */
+export default function dragDrop(tasks) {
+  const todoAppContainer = document.getElementById('todoAppContainer');
+  const draggables = document.querySelectorAll('.draggable');
+  draggables.forEach((draggable) => {
+    draggable.addEventListener('dragstart', () => {
+      draggable.classList.add('dragging');
+    });
+    draggable.addEventListener('dragend', () => {
+      draggable.classList.remove('dragging');
+      const todoItemsDiv = document.querySelectorAll('.item');
+      for (let i = 0; i < draggables.length; i += 1) {
+        tasks[(todoItemsDiv[i].id)].index = i;
+      }
+      tasks = tasks.sort((a, b) => (a.index - b.index));
+      localStorage.setItem('items', JSON.stringify(tasks));
+    });
+  });
```

Comment:
- [ ] After dragging something unusual happening, I have attached below screenshot where we can see clearly after `drag and drop` the the two task has the same `id`, because of this sometimes drag and drop is working and sometimes not. Please fix it. Make sure after dragging the `task` is assigning the correct `id`.

![Screenshot from 2021-07-22 03-24-44](https://user-images.githubusercontent.com/22434660/126565658-0cb4cbdf-2ce7-468d-a113-559a67c49fff.png)